src/main/java/net/neoforged/neoforge/registries/datamaps/builtin/Oxidizable.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.registries.datamaps.builtin;
+
+import com.mojang.serialization.Codec;
+import com.mojang.serialization.codecs.RecordCodecBuilder;
+import net.minecraft.core.registries.BuiltInRegistries;
+import net.minecraft.world.level.block.Block;
+
+public record Oxidizable(Block after) {
```

Comment:
Please document this class (link to the data map field, and explain what it is for)